scss/components/alert.scss

```diff
@@ -1,45 +1,66 @@
 @import "./../settings";
 @import "./../mixins";
-@import "./../functions";
 @import "./../icons/mixins";
-/*!
-.fd-alert+(--information, --success, --warning, --error, --no-icon)
-    .fd-alert__close
-*/
+
+@import "./../../theme-designer/Base/baseLib/sap_fiori_3/sass_variables.scss";
+@import "./../../theme-designer/Base/baseLib/sap_fiori_3/css_variables";
```

Comment:
Should wait for https://github.com/SAP/fundamental-styles/pull/239 to be merge and then import from `@sap-theming/theming-base-content`?

Reply:
this is done :)